tutorials/math/mathcoreGenVector.C

```diff
@@ -380,10 +380,10 @@ int testLorentzVector() {
    ok+= compare( q4.t(), q1.E(), "op E" );
 
    // test operator ==
-   XYZTVector        w1 = v1;
-   PtEtaPhiEVector   w2 = v2;
-   PtEtaPhiMVector   w3 = v3;
-   PxPyPzMVector     w4 = v4;
+   const XYZTVector&        w1 = v1;
+   const PtEtaPhiEVector&   w2 = v2;
+   const PtEtaPhiMVector&   w3 = v3;
+   const PxPyPzMVector&     w4 = v4;
    ok+= compare( w1 == v1, static_cast<double>(true), "== PxPyPzE");
```

Comment:
I think, here assign operator also was tested.
Therefore modified code is not equivalent to previous one.

Reply:
One probably should separate the testing code from the tutorials (which should show user the right way to use the classes)